### DISCUSSION TITLE
Welcome to conformal-prediction Discussions!

### DISCUSSION THREAD

**Discussion by aangelopoulos** [aangelopoulos/conformal-prediction#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Krasner** [comment]
Hi this is great work!
Just to clarify for classification tasks you would first do Platt scaling to optimize the calibration temperature. Then you would apply RAPS to the calibrate model to get the quantile threshold, correct?
Thanks!

**Reply by aangelopoulos** [reply]
You don't need to do Platt scaling first, but it can help get slightly better sets!

If you want to skip it, you still get the same guarantee.
